src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -217,7 +217,7 @@
   less jarring.
   */
   & code {
-    font-size: 0.6em;
+    font-size: 0.8em;
```

Comment:
Bumps to `0.8em`, as specific in [the task](https://app.asana.com/0/1202114367927919/1202439831418480/f)